src/examples/bus-journey-patterns/response.json

```diff
@@ -0,0 +1,2487 @@
+{
+  "id": "https://transportapi.com/v3/uk/bus/services/FPOT:25.json?app_id=<app_id>&app_key=<app_key>\u0026journey_patterns=true\u0026stop_geometry=true",
```

Comment:
This `\u0026` - is it some escaping problem? Is it a problem at all?

Reply:
Actually, I am not sure why not `&` directly. It shouldn't be a problem. I just copied the response JSON.